contrib/cf-remote/cf_remote/remote.py

```diff
@@ -67,6 +66,9 @@ def print_info(data):
     else:
         output["CFEngine"] = "Not installed"
 
+    if data["policy_server"]:
+        output["Policy server"] = data["policy_server"]
```

Comment:
What if we did `output["Policy server"] = data.get("policy_server")`? If there is none, there would be `None` in the output data and the output itself. Which makes sense to me, it's quite an important information.

Reply:
I prefer not printing it instead of None, but okay, I will change it.

Reply:
Fixed.